### PR TITLE
First pass at controller docs for Jekyll

### PR DESCRIPTION
**Pull request by GregHanson**

### CODE REVIEW

**Comment by GregHanson** [comment]
Controller currently does not have the same YAML support sidecar does - is there a timeframe on when this will be added, if at all?  Just wondering if the YAML stuff should at least be stubbed out or something with _TBD_  in that regard

File: _docs/controller/configuration-options.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+layout: page
+title: Configuration
+permalink: /docs/controller/controller-configuration-options/
+category: Controller
+order: 3
```
Comment (rshriram) [inline _docs/controller/configuration-options.md]: order: 1

File: _docs/controller/configuration-options.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+| | --version, -v | print the version | |
+{:.table .table-bordered .table-condensed .table-striped}
+
+## Example Configuration using Environment Variables
+
+```bash
+A8_LOG_LEVEL=debug
+```
+
+**Persistent Storage Backend:** Controller by default will use in-memory
```
Comment (rshriram) [inline _docs/controller/configuration-options.md]: How about a subsection?
"## Persistent Storage Backend"

File: _docs/controller/configuration-options.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+options, most of which are set through environment variables. The environment
+variables can be set via command line flags as well.
+
+The following environment variables are available. All of them are optional.
+
+| Environment Key | Flag Name                   | Description | Default Value |
+|:----------------|:----------------------------|:------------|:--------------|
+| A8_API_PORT | --api_port | API port | 8080 |
+| A8_CONTROL_TOKEN | --control_token | Controller API authentication token | ABCDEFGHIJKLMNOP |
+| A8_ENCRYPTION_KEY | --encryption_key | secret key | abcdefghijklmnop |
```
Comment (rshriram) [inline _docs/controller/configuration-options.md:19]: Needs more description.. 
e.g., secret key used to encrypt data stored in the database.

File: _docs/controller/configuration-options.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+installations. The Amalgam8 Controller supports a number of configuration
+options, most of which are set through environment variables. The environment
+variables can be set via command line flags as well.
+
+The following environment variables are available. All of them are optional.
+
+| Environment Key | Flag Name                   | Description | Default Value |
+|:----------------|:----------------------------|:------------|:--------------|
+| A8_API_PORT | --api_port | API port | 8080 |
+| A8_CONTROL_TOKEN | --control_token | Controller API authentication token | ABCDEFGHIJKLMNOP |
```
Comment (rshriram) [inline _docs/controller/configuration-options.md]: Description: API Auth token for control plane applications. Can view all tenant information with this token.

File: _docs/controller/configuration-options.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+| A8_ENCRYPTION_KEY | --encryption_key | secret key | abcdefghijklmnop |
+| A8_DATABASE_TYPE |  --database_type |	database type | memory |
+| A8_DATABASE_USERNAME | --database_username | database username | |
+| A8_DATABASE_PASSWORD | --database_password | database password | |
+| A8_DATABASE_HOST | --database_host | database host | |
+| A8_LOG_LEVEL | --log_level | logging level (debug, info, warn, error, fatal, panic) | info |
+| A8_AUTH_MODE | --auth_mode | Authentication modes. Supported values are: 'trusted', 'jwt'" | |
+| A8_JWT_SECRET | --jwt_secret | Secret key for JWT authentication | |
```
Comment (rshriram) [inline _docs/controller/configuration-options.md:26]: How does this work with multiple tenants? Might be useful to add a subsection after the persistent storage, on something like
"Setting up Controller in multi-tenant mode". And talk about the trusted, jwt auth mode, how each tenant (with multiple sidecars) will use their tokens, etc.

File: _docs/controller/configuration-options.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+{:.table .table-bordered .table-condensed .table-striped}
+
+## Example Configuration using Environment Variables
+
+```bash
+A8_LOG_LEVEL=debug
+```
+
+**Persistent Storage Backend:** Controller by default will use in-memory
+ storage, to persist created rules to a redis storage backend, enable
```
Comment (rshriram) [inline _docs/controller/configuration-options.md]: Also, while you are at this, break into two sentences please.

**Pull request merged**